bin/cloud-provision.sh

```diff
@@ -57,7 +58,10 @@ send_and_execute(){
 }
 
 copy_credentials() {
-    scp -r $OPENSTACK_CREDENTIALS_PATH ubuntu@$INSTANCE_IP:$JENKINS_HOME
+    # first novarc file in $OPENSTACK_CREDENTIALS_PATH
+    first_novarc=$(ls $OPENSTACK_CREDENTIALS_PATH/novarc* | sort -d | head -1)
```

Comment:
I have a directory .canonistack with novarc and novarc_lcy02. How would I select the lcy02?